crates/rpc/rpc-eth-api/src/helpers/block.rs

```diff
@@ -73,7 +73,7 @@ pub trait EthBlocks: LoadBlock {
                 return Ok(LoadBlock::provider(self)
                     .pending_block()
                     .map_err(Self::Error::from_eth_err)?
-                    .map(|block| block.body.len()))
+                    .map(|block| block.body.len()));
```

Comment:
please revert unrelated changes. we use the nightly formatter in reth `cargo +nightly fmt`. try command `make lint`.